### ISSUE TITLE
Tags/release versions

### ISSUE THREAD

**Issue by satta** [DataTables/DataTablesSrc#89]
Hi, I'm posting my request here because it is directly connected to GitHub. I have noticed that there is a bit of variety in the tagged version numbers here in this repository. As the Debian maintainer for DataTables, I am monitoring the version numbers tagged in this repo to detect new upstream versions released by the DataTables team for packaging.

However, recently a number of tags with version numbers >=3 (e.g. 3.2.2 -> https://github.com/DataTables/DataTablesSrc/releases/tag/3.2.2) have been pushed to this repo, the most recent of which all reference commit 6382fb4a7fb36d8546b03217613c746e527f06be:

![screen shot 2016-12-28 at 12 33 01](https://cloud.githubusercontent.com/assets/27732/21521093/ee205272-ccf9-11e6-8cf9-f641213af9b0.png)

This surprised me a bit, given that the version numbering scheme has been 1.10.x up to now. I was wondering whether these tags really reference a released version or whether they were tagged by accident?

**Comment by DataTables** [comment]
Tagged by accident. it appears that one of my scripts must be polluting this repo, or git has changed how tags are propagated up the file system (the extensions are inside an "extensions" repo, and these tags come from them) - they are completely wrong! Thanks for pointing this out!

I wouldn't normally delete tags, but these tags are very misleading, so I'd take that step on this occasion.

I will look int what is causing this issue and going to leave this issue open until I can do so (in the new year).

**Comment by satta** [comment]
Thanks for your quick reply and for confirming my suspicion. I will make a note to disregard these 'updates' and wait for the tags to be deleted.

**Comment by satta** [comment]
I'd like to kindly inquire if there has been any progress on this? I've noticed that the misleading tags are still there -- I would be happy to adjust my checking script to only consider the tags starting with 1.10.* if that's the naming scheme you will be sticking with for a while?

**Comment by DataTables** [comment]
Sorry about that. I thought I'd done it already, but it looks like it Might have done it for the distribution repos and not this one... It is now!

**Comment by satta** [comment]
Sorry to bother you about this, but I can still see the tags here in this repo today...

**Comment by DataTables** [comment]
One of my automated build scripts must be adding them for some reason... I'll check into it and figure out what is going on. Sorry about this.

**Comment by holtkamp** [comment]
Indeed, a ```composer update``` with empty caches and ```composer.json```:
```json
    "bower-asset/datatables.net": "@stable",
    "bower-asset/datatables.net-fixedheader-bs": "@stable",
```

now results in:

```
Composer\DependencyResolver\Operation\UpdateOperation
  - Updating bower-asset/datatables.net (1.10.13 => 2.1.1) Downloading: 100% Extracting archive
    REASON: Required by the root package: Install command rule (install bower-asset/datatables.net 1.10.13|install bower-asset/datatables.net 1.10.13|install bower-asset/datatables.net 2.1.1)
Composer\DependencyResolver\Operation\UpdateOperation
  - Updating bower-asset/datatables.net-bs (1.10.13 => 2.1.1) Downloading: 100% Extracting archive
    REASON: bower-asset/datatables.net-fixedheader-bs 3.1.2 requires bower-asset/datatables.net-bs >=1.10.9 -> satisfiable by bower-asset/datatables.net-bs[1.10.13, 2.1.1, dev-master].
```

**Comment by DataTables** [comment]
I'm having trouble removing the incorrect packages from npm and have a support request in with them about it.

**Comment by holtkamp** [comment]
Ok thanks for the update, standing by while monitoring https://github.com/npm/npm/issues/15521

**Comment by holtkamp** [comment]
@DataTables thanks, I tried to use:

```
    "bower-asset/datatables.net": "@stable",
    "bower-asset/datatables.net-bs": "@stable",
    "bower-asset/datatables.net-fixedheader-bs": "@stable",
```

This will cause ```2.1.1``` to be installed for ```datatables.net``` and ```datatables.net-bs```. I think the GitHub tags / releases for ```2.1.1`` need also be removed?

- https://github.com/DataTables/Dist-DataTables/releases
- https://github.com/DataTables/Dist-DataTables-Bootstrap/releases

And, (outside this example) also:
- https://github.com/DataTables/Dist-DataTables-Bootstrap4/releases
- .... others?

**Comment by DataTables** [comment]
I think we are finally good now. Some of the incorrect tags are still in place (I'm bothered about deleting them after such a long time), but the latest release is 1.10.15 and bower and npm are seeing that as being the case.

**Comment by holtkamp** [comment]
Ok, nice thanks!

Will use this for now: 

```json
"bower-asset/datatables.net": "^1.10",
"bower-asset/datatables.net-bs": "^1.10",
"bower-asset/datatables.net-fixedheader-bs": "^3.1"
```
